File: BOJ/10000-10999/10818.cpp
#include <iostream>

using namespace std;

int main() {
	int N;
	cin >> N;

	int maxVal = -1000001;
	int minVal = 1000001;
	for (int i = 0; i < N; i++) {
		int x;
		cin >> x;
		maxVal = max(maxVal, x);
		minVal = min(minVal, x);
	}

	cout << minVal << " " << maxVal;
}

File: BOJ/10000-10999/10818.swift
import Foundation

let N = Int(readLine() ?? "") ?? 0
let arr = readLine()!.split(separator: " ").map{ Int(String($0))! }

var min = 1000001
var max = -1000001
for i in 0..<N{
    if min > arr[i]{
        min = arr[i]
    }
    if max < arr[i]{
        max = arr[i]
    }
}

print("\(min) \(max)")
